packages/serverpod/test/database/database_query_test.dart

```diff
@@ -10,6 +10,43 @@ class _TableWithoutFields extends Table {
   List<Column> get columns => [];
 }
 
+class _TableWithManyRelation extends Table {
+  final String _relationAlias;
+  _TableWithManyRelation(
+      {required String relationAlias,
+      required super.tableName,
+      super.tableRelation})
+      : _relationAlias = relationAlias;
+
+  ManyRelation<_TableWithManyRelation>? _manyRelation;
+  ManyRelation<_TableWithManyRelation> get manyRelation {
+    if (_manyRelation != null) return _manyRelation!;
+
+    var relationTable = createRelationTable(
+      relationFieldName: _relationAlias,
+      field: id,
+      foreignField: id,
```

Comment:
Question: Is this really correct? why do we set the same field to both sides here?